pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/block/BlockListener.kt

```diff
@@ -486,6 +488,31 @@ internal object BlockListener : Listener {
         }
     }
 
+    @EventHandler
+    private fun onPlayerToggleSneak(event: PlayerToggleSneakEvent) {
+        val block = event.player.location.add(0.0, -1.0,0.0).block
+        val pylonBlock = BlockStorage.get(block)
+        if (pylonBlock is PylonSneakableBlock) {
+            /*
+            * Event player is from before the event is triggered, so when the player
+            * is marked as *not* sneaking, they just toggled it.
+            * */
+            if (!event.player.isSneaking)
+                pylonBlock.onSneakStart(event)
+            if (event.player.isSneaking)
+                pylonBlock.onSneakEnd(event)
```

Comment:
```suggestion
            if (!event.player.isSneaking) {
                pylonBlock.onSneakStart(event)
            } else {
                pylonBlock.onSneakEnd(event)
            }
```
We put braces on almost everything except for the occasional `break`/`continue`/`return`